pulsar-client-cpp/python/pulsar_test.py

```diff
@@ -486,6 +486,23 @@ def test_consumer_is_connected(self):
         self.assertFalse(consumer.is_connected())
         client.close()
 
+    def test_consumer_get_last_message_id(self):
+        client = Client(self.serviceUrl)
+        topic = "test_consumer_get_last_message_id"
+        sub = "sub"
+        consumer = client.subscribe(topic, sub)
+
+        producer = client.create_producer(topic)
+
+        for i in range(10):
+            producer.send(b"hello-%d" % i)
+            message_id = consumer.get_last_message_id()
```

Comment:
we should add a test with a empty topic

possibly we should cover the case of a truncated topic, but I am not sure we have a API to truncate a topic in python. not a blocker

Reply:
Sure, I can cover that test too. I need to address the underlying C++ first, though.